packages/cli/src/commands/new/index.ts

```diff
@@ -9,15 +9,11 @@ export function New(context: IContext): CommandModule<{}, {}> {
   return {
     command: 'new',
     aliases: 'n',
-    describe: 'create a new app project',
+    describe: 'create a new app project. You can set the default language with the "set-lang" command.',
```

Comment:
NIT: if lang is set, this can show "typescript" or "csharp". If it isn't, then it can show this help text?